### ISSUE TITLE
streamdeck -n still launches the full UI in Fedora 37 

### ISSUE THREAD

**Issue by tdcam** [timothycrosley/streamdeck-ui#279]
**Describe the bug**
I have reinstalled streamdeck-ui, and added a startup command to launch streamdeck without the UI. I added /home/thomas.cameron/.local/bin/streamdeck -n to my session and startup tool under Xfce.

When I log in, it launches the full UI. I'd like it to only launch the app so that it's minimized in the tool bar.

**To Reproduce**
1. Install streamdeck-ui
2. launch streamdeck -n from the command line or from autostart
3. The full UI is launched
4. I expect the UI to NOT be launched

**Expected behavior**
Launch streamdeck-ui without the configuration UI, just minimized to the system tray.

**Screenshots**
![Screenshot_2022-11-29_10-06-19](https://user-images.githubusercontent.com/63890286/204581219-b066b931-eb54-489b-ab68-091fc4d2a338.png)

**Operating system (please complete the following information)**
 - OS: Fedora 37
 - Desktop environment: Xfce
 - Python version python3-3.11.0-1.fc37.x86_64

**Stream Deck Version**
 - XL

**Comment by Data-007** [comment]
Hi
If memory servers streamdeck command should be on your path
So try making  small script or  alter your command to "streamdeck -n" minus the "".
 
A simple test  quit streamdeck from the tray.
Launch cli
type streamdeck -n and press enter.

If that works just alter your script/program kickoff.

HIH if you have further problems or need a script Ile post mine.

--
Steve

**Comment by Data-007** [comment]
Hi
XXXXXXXXXXXXScriptXXXXXXXXXXXXXX
#!/usr/bin/zsh
PATH=$PATH:$HOME/.local/bin
python ~/.local/bin/streamdeck -n

**Comment by Data-007** [comment]
Hi
Just been trying to recreate your issue, and I failed to install fedora. Ive ended up with Gnome I think, how anyone can use deal with that I don't know.
Sorry, I can't do it. IT being recreated the problem.
--
Steve

**Comment by andersonid** [comment]
Same here. Fedora 37, gnome, X11.
I cant install pyside2, and shiboken2 versions 5.15.  PIP only installs 5.13... Maybe something related to it.

**Comment by dodgyrabbit** [comment]
@tdcam and others - from the screenshot I can tell that you have an old version installed which does not support the -n option.
The solution is to get to the latest version 2.0.13.

Have a look here: https://timothycrosley.github.io/streamdeck-ui/docs/installation/fedora/

**Comment by tdcam** [comment]
For what it's worth, I re-ran setup a couple of different times and never got it to work. So instead, what I did was to run:

_pip uninstall streamdeck-ui hidapi pillow pynput pyside2 streamdeck evdev shiboken2_

That uninstalled all the stuff under ~/.local, and then I was able to re-run 

_python -m pip install streamdeck-ui --user_

Now, it's working as expected. I can pass the -n argument to streamdeck and it pops up in my system tray as expected, without having to close the window. 

Thanks for the  pointer!
